README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
```

Comment:
Flowchart is mostly OK, although you do not actually decrement the number of guesses remaining anywhere. Important to make this kind of thing explicit.